staging/src/k8s.io/apiserver/pkg/endpoints/groupversion.go

```diff
@@ -71,9 +70,6 @@ type APIGroupVersion struct {
 	// version (for when the inevitable meta/v2 group emerges).
 	MetaGroupVersion *schema.GroupVersion
 
-	// RootScopedKinds are the root scoped kinds for the primary GroupVersion
-	RootScopedKinds sets.String
```

Comment:
This is a breaking change, but maybe it doesn't matter because it's literally not used anywhere?

@jpbetz